userdocs/diagnostics/protocol-type-non-conformance.md

```diff
@@ -1,46 +1,59 @@
-# Protocol type not conforming to itself
-Protocols in Swift may be used as types. Protocols as types are sometimes called existential types.
+# Protocol Types Cannot Conform to Protocols
 
+In Swift, a protocol that does not have `Self` or associated type requirements can be used as a type. You can use a variable or constant of a protocol type, also called an __existential type__, to hold a value of any conforming type:
 
 ```swift 
-protocol P {}
-
-struct S: P {}
+protocol Animal {
+    func makeNoise()
+    static var species: String { get }
+}
+struct Dog: Animal {
+    func makeNoise() { print("Woof") }
+    static var species: String = "Canus familiaris"
+}
+struct Cat: Animal {
+    func makeNoise() { print("Meow") }
+    static var species: String = "Felis catus"
+}
 
-var s: P = S() // This creates existential type because the protocol P is used as a type
+var animal: Animal // `Animal` is used here as a type.
+animal = Dog()
+animal.makeNoise() // Prints "Woof".
+animal = Cat()
+animal.makeNoise() // Prints "Meow".
 ```
 
-However, a protocol type does not conform to protocols - not even the protocol itself. 
-Allowing existential types to conform to protocols is unsound. For protocols with static method, initializer, or associated type requirements, the implementation of these requirements cannot be accessed from the protocol type - accessing these kinds of requirements must be done using a concrete type.
-
-Let's walk through the example below:
+Notice that it is possible to invoke the method `makeNoise()` on a value of type `Animal`, just as it is possible to do so on a value of concrete type `Dog` or `Cat`. However, the static property `species` is not available for the existential type:
 
 ```swift
-protocol Word: Hashable {
-    var word: String { get }
-}
+print(Dog.species)    // Prints "Canus familiaris"
+print(Cat.species)    // Prints "Felis catus"
+print(Animal.species) // Error
+```
 
-struct Singular: Word {
-    var word: String
-}
+Since a type conforms to a protocol only when it satisfies _all_ of that protocol's requirements, the existential type `Animal` does not conform to the protocol `Animal` because it cannot satisfy its own requirement for the static property `species`:
```

Comment:
```suggestion
Since a type conforms to a protocol only when it satisfies _all_ of that protocol's requirements, the existential type `Animal` does not conform to the protocol `Animal` because it cannot satisfy the protocol's requirement for the static property `species`:
```

^ This is really nitpicky, but I think it emphasizes the difference between the existential and the protocol